awx_collection/plugins/module_utils/controller_api.py

```diff
@@ -61,12 +62,14 @@ class ControllerModule(AnsibleModule):
         'username': 'controller_username',
         'password': 'controller_password',
         'verify_ssl': 'validate_certs',
+        'req_timeout': 'request_timeout',
```

Comment:
oh, we do process this from env vars and config file..

Reply:
this I was not sure about, willing to change, but everything refers to req_timeout,

Reply:
This turned out to be an issue just changed to request_timout everywhere.